serg/mtr-bash-completion.sh

```diff
@@ -1,30 +1,30 @@
 _mtr_complete_testnames ()
 {
-  dir=$1
-  [ -d $dir/t ] && dir=$dir/t
-  testnames=$( cd $dir && echo *.test | sed -e 's/\.test\>//g' )
+  dir="$1"
+  [ -d "$dir"/t ] && dir="$dir"/t
+  testnames=$( cd "$dir" && echo *.test | sed -e 's/\.test\>//g' )
 }
```

Comment:
did you actually test it with a path that contains spaces? Or you've quoted paths just in case, without testing?